### DISCUSSION TITLE
Tables are too vanilla and need improvement

### DISCUSSION THREAD

**Discussion by kuna1bajaj** [outline/outline#6910] [Ideas]
The tables in Outline seem very vanilla. 
- Can I hold the column sizes?
- Can I select types of fields in a dropdown?
- Can we have types of columns like in notion or Airtable (without having to insert an Airtable itself)

This would be a great help

**Comment by hilsonp** [comment]
1. Being able to shrink column size (when they hold only 1 or 2 digits, the default size is eating too much page real estate).
2. Also, being able to choose background color for the cells would be nice:

- Declare a heading row so that it has a background color (or not)
- Declare a heading column so that it has a background color (or not)
- Put an arbitrary background color on a cell

3. Be able to merge some cells (or split some, it depends the way you look at it ;-) ) is often useful.

**Comment by matthiscock** [comment]
Hi All

Just to add my opinions on this.
For me the tables are basic, but that can be a good thing.
A few additions would make them far more attractive and powerful

Colour a row
- this enables simple highlighting

Colour a cell
- same

Dropdowns
- This would be where you can define a drop down.  Some could be user based.
- Would take you close to airtable functionality without huge complexity

(Phase 2) Ordering
- This does add complexity, but would then put you in the realms of task management - like monday.com etc

**Comment by pcxeon2** [comment]
Hi All,

I just discovered outline fews days ago and this project is really awesome!
But like previous comments, unfortunately I feel the table not usefull without sort, filter features.

Is it plan?

**Comment by ghost** [comment]
Hey everyone! I’ve started a new discussion on adding **structured data (databases)** to Outline and referenced this thread among other related ideas.

The new post is quite visual and includes a current ecosystem overview, potential features, and mockups. I’d love your feedback:
⭐ #8476
